app/components/EditEventModal.tsx

```diff
@@ -51,6 +50,26 @@ const EditEventModal = ({ event }: { event: SerializableEventWithTags }) => {
     })();
   };
 
+  const deleteEventHandler = async () => {
+    if (confirm(`Are you sure you want to delete ${event.title}?`)) {
+      const response = await fetch(`/api/delete-event?id=${event.id}`, {
+        method: 'DELETE',
+      });
+    
+      if (!response.ok) {
+          console.error("Failed to delete event:", response);
+          alert(`Failed to delete the event: "${event.title}".`);
+          return;
+      }
```

Comment:
A little nitpicking: has this case been tested? In what case will this response be not ok? If it has been tested, also include the possible situation in your PR description. Otherwise maybe point out here that for now the response always returns ok

Reply:
There would be couple of cases when the response is not ok: network error / server-side issues / incorrect endpoint / unauthorized access, etc (I asked chatgpt about this btw). 

The fact that the delete works shows that there are no server-side issues nor incorrect endpoint (correct me if im wrong). But I have not tested the case when an unauthorized person tries to delete an event. How would I usually test this? Do I need a random mit student's account who is not authorized to delete events?

Reply:
You can test the allowDelete function, but @lad6558 it seems that this repository does not have any tests?

I think it is always worth it to manually check that people who shouldn't have access aren't allowed to perform operations. Sometimes even large companies forget

Reply:
Yeah makes sense - if it is difficult to test, no need to actually test it. I would prefer the fact that this error case has not been tested to be in the comment, but that is a rather minor style preference. 

@gabrc52 Yeah I am talking about manual testing. Thanks for the comment